### PR TITLE
spring-security-crypto: initial integration

### PR DESCRIPTION
**Pull request by onionpsy**

### CODE REVIEW

File: projects/spring-security/BCryptPasswordEncoderFuzzer.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
```
Comment (Navidem) [inline projects/spring-security/BCryptPasswordEncoderFuzzer.java:1]: Copyright header is missing.

**Comment by Navidem** [comment]
How about `project.yaml` for initial integration?

**Comment by onionpsy** [comment]
> How about `project.yaml` for initial integration?

`spring-security-crypto` is part of `spring-security` which is already onboarded and therefore contains the yaml file

File: projects/spring-security/BCryptPasswordEncoderFuzzer.java (hunk 1 above)
Comment (onionpsy) [reply projects/spring-security/BCryptPasswordEncoderFuzzer.java:17]: Added it

**Pull request merged**